modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -52,6 +52,8 @@ const (
 	metricRequestClientSeconds = "traces_service_graph_request_client_seconds"
 )
 
+const virtualNodeLabel = "__virtual_node"
```

Comment:
I asked for this. Should we drop it? Just trying to avoid conflicts with actual span attributes.

Reply:
I'll be OK with and without the `__` prefix

Reply:
OK - then to not raise any questions to future readers, and because it seems a collision is unlikely, I'll leave it as **without** the `__` prefix.

It'd be good in the future to identify and sanitize in the same way as in spanmetrics, happy to work on that small improvement in a future PR if you need me.